### DISCUSSION TITLE
CNN引用照片指责「中方代表未在泽连斯基讲话时起立」，但图中所谓「中方代表」实为越南副总理。请问西方媒体后续会报道真相吗？

### DISCUSSION THREAD

**Discussion by xyzhou-1** [The-Run-Philosophy-Organization/run#418] [General]
这不添加到合订本里？

**Comment by ghost** [comment]
给个新闻链接？

**Comment by mancoffee** [comment]
求新闻链接，谢谢

**Comment by xyzhou-1** [comment]
但凡你会用搜索引擎？
英文搜不到吗，那也理解，这就是信息茧房吗
![image](https://user-images.githubusercontent.com/47747466/170185874-ff83016e-8270-4203-9b30-338e1af6b428.png)
四个越南人，没一个中国人，笑死了，如果cnn是想说越南是中国领土，那我也没意见
反正内宣目的也达到了，美国人也不需要知道真相——他们只需要继续相信中国是大魔王就行了。

**Reply by ghost** [reply]
> 但凡你会用搜索引擎？ 英文搜不到吗，那也理解，这就是信息茧房吗 ![image](https://user-images.githubusercontent.com/47747466/170185874-ff83016e-8270-4203-9b30-338e1af6b428.png) 四个越南人，没一个中国人，笑死了，如果cnn是想说越南是中国领土，那我也没意见 反正内宣目的也达到了，美国人也不需要知道真相——他们只需要继续相信中国是大魔王就行了。

我就懒得搜索而已，而且

[英美主流媒体盘点：为什么我不建议你看 VOA，CNN](https://blog.shuziyimin.org/587)

不知道这篇文章您是否阅读过（

**Reply by xyzhou-1** [reply]
挺好，希望你也多推荐给美国人看这篇文章，CNN的受众又不是我。这篇文章也不妨碍这则新闻构成信息茧房的事实，因为不会有美国媒体来澄清

**Reply by walker1996** [reply]
BBC

BBC 英国广播公司，在是英国一家独立运作的公共媒体，资金主要来自于英国国民所缴纳的电视牌照费，英国政府拨款仅占较少一部分。

资助方：公共资费
立场倾向：中立
可信度：高

BBC针对除中国外的新闻可信度高不高我不知道，但针对中国的真心可信度不高（这两年）。
